docs/guides/endgame.md

```diff
@@ -21,7 +21,7 @@
     - [Publishing steps](#publishing-steps)
   - [Patch releases](#patch-releases)
     - [Publishing steps](#publishing-steps-1)
-  - [Release candidate releases](#release-candidate-releases)
+  - [Release Candidate releases](#release-candidate-releases)
```

Comment:
```suggestion
  - [Release Candidate releases](#release-candidate-releases)
```
Following a quick look-about, it seems like this isn't a proper noun. Is there a reason it's now capitalized?

Reply:
It happened automatically when I ran doctoc for some reason, it might have been my prettier or something. I can revert it!

Reply:
It's generated from the header! If you update it there, should be good 😄